src/WebUI/Common/KubeSummary.tsx

```diff
@@ -357,7 +358,7 @@ export class KubeSummary extends React.Component<IKubeSummaryProps, IKubernetesC
 
         // show cluster if available, and show namespace only if namespaces list is not available
         return (
-            <div className="flex-column rhythm-vertical-8">
+            <div className={css("flex-column rhythm-vertical-8", this.props.clusterUrl && "cluster-link")}>
```

Comment:
we should not do this.

in the function, _getHeaderClusterNameComponent() embed Link with span. so you will not see the problem you have. Also when i use keyboard i could see the following:

![image](https://user-images.githubusercontent.com/26214977/61456671-46cb0c00-a984-11e9-8dd6-80597187901f.png)

you can see that the link does not have a wrapper, for this add a new class with "flex-column rhythm-vertical-8", and have something like

.some-new-class>:last-child {
    margin-bottom: 4px
}

so that you can see the border properly.

also please check with PM that they want to make this link as subtle. i am keeping this on hold, as the change making it to display: block, not the right fix at sub-header level.